README.md

```diff
@@ -1,2 +1,34 @@
-## Dev-Mountain Solidity Contract
+# **Dev. Mountain Tech Festival(Thailand) 2022 - Solidity Contract**
 
+
+## What is Dev. Mountain Tech Festival  
+
+Dev. Mountain Tech Festival is Thailand tech events that setup at
+18-20 March 2022 Toscana Valley Khao Yai.This event is have many top-tier Thai developers as speaker and have many different topic for developers or anyone who interested in technology
+in additional, this tech event also has blockchain hackathon inside it
+this Hackathon is having 5 team that passing the first round before can participate in the main event .
```

Comment:
ตรงนี้ขอเน้นช่วงแข่งออนไลน์และเน้นย้ำว่ามีทีมแค่ 5 ทีมผ่านเข้ามาเพิ่มหน่อย เพื่อความเท่ แล้วก็ขออธิบายเป้าหมายของงานเพิ่มนิดนึง เพราะข้างบนเรายังไม่ได้พูดถึงว่า hack นี้ทำอะไร นอกจากบอกว่าเป็น blockchain hack, ทำทำไม และ impact อยู่ที่ไหน
In this hackathon, participants have to pass an online smart contract competition. Finally, there were 5 teams participating in the main event and facing 5 unique domains, which come from different businesses. The owner of each problem also entered and waited to hear how competitors use blockchain to solve their problems effectively in this event.